Dockerfile

```diff
@@ -7,33 +17,16 @@ ENV AR_LISTEN_UDP_ACARS=5550 \
     AR_SERVE_TCP_ACARS=15550 \
     AR_SERVE_TCP_VDLM2=15555 \
     AR_SERVE_ZMQ_ACARS=45550 \
-    AR_SERVE_ZMQ_VDLM2=45555 \
-    PATH=/opt/acars_router:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-
-COPY acars_router/requirements.txt /opt/acars_router/requirements.txt
+    AR_SERVE_ZMQ_VDLM2=45555
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-
+COPY rootfs /
+COPY --from=builder /tmp/acars_router/target/release/acars_router /opt/acars_router
+# hadolint ignore=DL3008,DL3003,SC1091
 RUN set -x && \
-    TEMP_PACKAGES=() && \
-    KEPT_PACKAGES=() && \
-    TEMP_PACKAGES+=(build-essential) && \
     apt-get update && \
-    apt-get install -y --no-install-recommends \
-        "${KEPT_PACKAGES[@]}" \
-        "${TEMP_PACKAGES[@]}" \
-        && \
-    python3 -m pip install --no-cache-dir --upgrade pip && \
-    python3 -m pip install --no-cache-dir --requirement /opt/acars_router/requirements.txt && \
-    # Clean up
-    apt-get remove -y "${TEMP_PACKAGES[@]}" && \
-    apt-get autoremove -y && \
-    rm -rf /src/* /tmp/* /var/lib/apt/lists/* && \
+    apt-get install -y --no-install-recommends libzmq3-dev && \
```

Comment:
Probably yes. I forget why I left it there. As I recall it needed the ZMQ libraries still installed in the system and uninstalling that package removed them.

Looking at https://packages.debian.org/bullseye/libzmq3-dev it seems like [libzmq5](https://packages.debian.org/bullseye/libzmq5) is installed as part of that package so perhaps installing that separately and keeping it will work. I'll play with it.